core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala

```diff
@@ -192,6 +192,10 @@ object TiUtil {
       tiConf.setTikvRegionSplitSizeInMB(conf.get(TiConfigConst.TIKV_REGION_SPLIT_SIZE_IN_MB).toInt)
     }
 
```

Comment:
No setting "USE_TIFLASH"? We may want to use tiflash for various usage in our local environment, not only for testing.